iosHackthonApp/Modules/Features/Login/Sources/Login/LoginViewModel.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  File.swift
+//  
+//
+//  Created by Raynelle Francisca on 26/11/2020.
+//
+
+import Foundation
+import SwiftUI
+import shared
+import Components
+import Theming
+
+public final class LoginViewModel: ObservableObject {
+
+	@Published var email: String = ""
+	@Published var password: String = ""
+	@Published var showingAlert = false
+	@Published var activeAlert: ActiveAlert = .collection
```

Comment:
I don't think you need this.  It was my way of being able to show different alerts on the same screen but there is only one alert so I think we can get rid of it

Reply:
yup..deleted